.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -1,13 +1,15 @@
 name: E2E tests ondemand
 
 on:
-  workflow_dispatch:
   schedule:
     # At 03:00 UTC on Monday, Wednesday, and Friday.
     - cron: 0 3 * * 1,3,5
-
-env:
-  branch: release-1.4
```

Comment:
@StefanHauth only change this line please, and you can research if you want separately.